### PR TITLE
Hey, it's kind of a website!

### PR DESCRIPTION
**Pull request by designatednerd**

### CODE REVIEW

**Comment by designatednerd** [comment]
@vokal/webandsystems HAY GUYZ I MADE A WEBSITE SORT OF. 

Anyone wanna take a look at this and tell me exactly how dumb I'm being about doing this? Lots of library stuff, since it all has to be local for extensions.

File: ext/src/ThemePlayground.html [hunk 1]
```diff
@@ -0,0 +1,640 @@
+<!DOCTYPE html>
```
Comment (foresmac) [inline ext/src/ThemePlayground.html:1]: Just delete this file.

**Comment by jrit** [comment]
You could, if you wanted to, use a package manager like bower and then you just commit a reference to the package instead of the package itself. Given you are probably mostly just trying to get this working here, that may not be your top priority. But that is how we usually deal with all the library junk.

**Comment by foresmac** [comment]
Yeah, I would have just committed all the lib stuff first, and then actual code second. Kind of like how every single iOS project has a first commit that is like 45 files of boilerplate crap.

**Comment by foresmac** [comment]
But before my scroll buffer broke, everything LGTM.

**Comment by designatednerd** [comment]
@foresmac I solemnly swear I will open a seperate pull when I get around to adding `chart.js` tomorrow.

**Comment by foresmac** [comment]
@designatednerd Sounds like a plan. Might have @jrit show you how to user bower for this stuff, though, makes things pretty tight.

**Comment by designatednerd** [comment]
Maybe during some after-hack days cleanup once I get the fucking thing working :)

**Comment by Tathanen** [comment]
I think Chris's gifs pretty much sum it up.

**Comment by designatednerd** [comment]
OK, merging this on @foresmac's LGTM.

**Pull request merged**